bin/mff2json.py

```diff
@@ -12,38 +18,48 @@
 "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 ANY KIND, either express or implied.
 """
+from os.path import splitext, sep, join
+from mffpy import Reader, Writer
+from argparse import ArgumentParser
 
-"""
-This script is used to export the contents of an MFF file into a JSON format.
+import glob
 
-@author: Damian Persico - persicodamian@gmail.com
-"""
 
-from os.path import splitext, sep
-from mffpy import Reader, Writer
-from argparse import ArgumentParser
+def mff2json(input_path):
+    """
+    This function undertakes the work of effectuating the conversions of a passed MFF file, or set of MFF files into
+    a JSON format.
 
+    These JSON formatted conversions will be saved in the same directory as the MFF(s) converted.
+    :param input_path: A string describing a path to a specific MFF, or a collection of them.
+    :return:
+    """
 
-def mff2json(input_filename):
-    if input_filename:
+    file_list = []
+
+    if '.mff' in input_path:
+        file_list = [input_path]
+    else:
+        file_list = glob.glob(join(input_path, '*.mff'))
+
+    for file in file_list:
         # Check .mff input
-        dir_and_base, ext = splitext(input_filename.rstrip(sep))
-        assert ext.lower() == '.mff', f"{input_filename} is not a valid .mff directory"
+        dir_and_base, ext = splitext(file.rstrip(sep))
+        # assert ext.lower() == '.mff', f"{input_path} is not a valid .mff directory"
         output_filename = dir_and_base + '.json'
 
         # Read data from an MFF file
-        reader = Reader(input_filename)
-        print("Reading data from " + str(input_filename))
+        reader = Reader(file)
```

Comment:
what happens if `file` is an invalid mff?

Reply:
After some prompted digging. `Reader` throws a ValueError if an invalid file is passed to it.  Since I had to dig, took the opportunity to add a note on the throw to the `reader.py` file - as the actual throwing of `ValueError` is a property that apparently comes along from the `mffdir` class.